docs/guides/key-manager/upgrade-lsp6.md

```diff
@@ -0,0 +1,246 @@
+---
+sidebar_label: 'Upgrade LSP6 Key Manager'
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade the LSP6 Key Manager
+
+:::info Requirements
+
+You will need a Universal Profile that you can control via its KeyManager to follow this guide.
+If you don't have a Universal Profile yet, follow our previous guide [**Create a Universal Profile**](../universal-profile/create-profile.md) or look at the [_lsp-factory.js_](../../tools/lsp-factoryjs/deployment/universal-profile.md) docs.
+
+:::
+
+In this guide, we will learn how to upgrade the LSP6 Key Manager of your Universal Profile to the latest version available.
+
+By the end of this guide, you will know how to:
+
+- Deploy a new LSP6 Key Manager with the last updates.
+- Transfer ownership of your Universal Profile through the old LSP6 Key Manager.
+- Accept ownership of your Universal Profile through the new LSP6 Key Manager
+
+## Setup
+
+```shell
+npm install @lukso/lsp-smart-contracts web3
+```
+
+## Step 1 - Set up constants
+
+Create a `constants.js` file with the following constants
+
+- `privateKey`: private key of a controller address
+- `keyManagerAddress`: address of the current LSP6 Key Manager
+- `universalProfileAddress`: address of your Universal Profile
+  This controller must have the CHANGEOWNER permission
+
+After this first step you can save & close `constants.js`.
+
+```js
+export const privateKey = '0x...';
+export const universalProfileAddress = '0x...';
+export const keyManagerAddress = '0x...';
+```
```

Comment:
I don't think it is necessary to split things in multiple files.

These guides are about showing how to do things, it is ok to duplicate and to not have a clean code.

By putting these 3 lines, without export, on the main file, you make the tutorial easier to read.

Also the final code will be easier as users can simply "copy/paste" and they don't have to create another file just to save 3 variables in them.